src/main/java/it/gov/pagopa/pu/organization/repository/BrokerRepository.java

```diff
@@ -2,9 +2,16 @@
 
 import it.gov.pagopa.pu.organization.model.Broker;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 
+import java.util.Optional;
+
 @RepositoryRestResource(collectionResourceRel = "broker", path = "brokers")
 public interface BrokerRepository extends JpaRepository<Broker,Long> {
 
+  @Query("select b from broker b join Organization o on b.organizationId = o.organizationId " +
```

Comment:
this is not right query!
This way you are retrieving the broker organization, while you should retrieve the organization's broker :)

Have you tested that the join written ASIS is working (or have you started locally the application and executed the API to verify if there is error or not)?
```suggestion
  @Query("select b from broker b join Organization o on b.brokerId = o.brokerId " +
```